src/main/resources/db/migration/postgresql/V1_8__2055_Topic_Column_On_Workstack_Team_Queue.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE VIEW stage_data AS
+    SELECT c.reference AS case_reference, c.type AS case_type, c.data as data, c.created AS case_created, s.*, t.text
```

Comment:
```suggestion
    SELECT c.reference AS case_reference, c.type AS case_type, c.data as data, c.created AS case_created, s.*, t.text AS case_assigned_topic
```

Then use this above?